src/lib.rs

```diff
@@ -747,6 +747,24 @@ impl<'a, 'b> ExpandedName<'a, 'b> {
     }
 }
 
+impl ExpandedName<'static, 'static> {
+    /// Create a new instance from static data.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use roxmltree::ExpandedName;
+    /// const DAV_HREF: ExpandedName =
+    ///     ExpandedName::from_static(("urn:ietf:params:xml:ns:caldav:", "calendar-data"));
+    /// ```
+    pub const fn from_static(v: (&'static str, &'static str)) -> Self {
```

Comment:
I know I am late to the game, but if this is not yet released, I would like to propose to drop the pair here because it does not serve a purpose as the function could also take two arguments with self-describing names, e.g.

```rust
pub const fn from_static(urI: &'static, name: &'static str) -> Self {
  Self {
    uri: Some(uri),
    name,
  }
}
```

If we want to make this properly match the internals, we might even want to accept `uri: Option<&'static str>`.

@RazrFalcon Would you accept a follow-up PR doing one or both changes?

Reply:
>If it has to be called explicitly in any case, why force everyone to write an extra set of parentheses?

API consistency?

Either-way, I don't mind switching to two arguments. So PR is welcome.